src/neo4j_genai/generation/prompts.py

```diff
@@ -147,9 +185,32 @@ class ERExtractionTemplate(PromptTemplate):
 
 Input text:
 
-{text}
+{query_text}
 """
-    EXPECTED_INPUTS = ["text"]
+    EXPECTED_INPUTS = ["query_text"]
 
-    def format(self, text: str, schema: dict[str, Any], examples: str) -> str:
-        return super().format(text=text, schema=schema, examples=examples)
+    def format(
+        self,
+        schema: dict[str, Any],
+        examples: str,
+        query_text: str = "",
+        text: Optional[str] = None,
```

Comment:
After discussing with the team, we see no real reason to rename this parameter at this stage. If that's important to you, may I suggest you open another PR with this change so that we can discuss this specifically? Thank.

Reply:
That's ok. I'll change this back.